src/libLookup/Lookup.cpp

```diff
@@ -2934,7 +2934,7 @@ bool Lookup::ProcessSetMinerInfoFromSeed(
 bool Lookup::ProcessSetTxBlockFromSeed(
     const zbytes& message, unsigned int offset, const Peer& from,
     [[gnu::unused]] const unsigned char& startByte) {
-  //#ifndef IS_LOOKUP_NODE
+  // #ifndef IS_LOOKUP_NODE
```

Comment:
plese remove

Reply:
It looks like these are Artems placeholders marked as for later.